articles/client-platforms/socket-io/01-login.md

```diff
@@ -12,9 +12,9 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
```

Comment:
@mehreencs87 Please add the following line:
`githubUrl: 'https://github.com/auth0-samples/auth0-socket.io-samples/tree/master/00-Starter-Seed'`

See the updated [contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects) for details.